### PR TITLE
Better separation for inspect package by directory

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: lib/pana.dart [hunk 1]
```diff
@@ -46,38 +44,32 @@ class PackageAnalyzer {
...
+  }
 
-    Version ver;
-    if (version != null) {
-      ver = new Version.parse(version);
-      log.info("Version: $ver");
-    }
+  Future<Summary> inspectDir(String packageDir,
+      {bool keepTransitiveLibs: false}) async {
+    _logEnv();
```
Comment (kevmoo) [inline lib/pana.dart]: The next call to `_inspect` also calls `_logEnv` – redundant?

File: lib/pana.dart [hunk 2]
```diff
@@ -46,38 +44,32 @@ class PackageAnalyzer {
...
     String version,
-    String packageDir,
     bool keepTransitiveLibs: false,
   }) async {
-    var toolProblems = <ToolProblem>[];
-    var sdkVersion = _dartSdk.version;
-    log.info("SDK: $sdkVersion");
-
-    log.info("Package: $package");
+    _logEnv();
```
Comment (kevmoo) [inline lib/pana.dart]: Also ends up calling _inspect – perhaps _logEnv should just be within _inspect...

File: lib/pana.dart (hunk 1 above)
Comment (isoos) [reply lib/pana.dart]: Good catch! The intent here is to log env information before starting to download the package, and that's reason it is duplicated in both method. Shall I just move both `_inspect`?

Comment (kevmoo) [reply lib/pana.dart]: Well, it's already there 😄

Comment (isoos) [reply lib/pana.dart]: done

**Pull request merged**